### DISCUSSION TITLE
Route Plugin from- class

### DISCUSSION THREAD

**Discussion by mocodev** [swup/swup#620] [Help] [answered]
Is there a setting I am missing to enable the `from-` class to be added to the html element  once content is replaced? It seems as if the route plugin is only adding `from-` and `to-` classes when leaving. It would be useful if `from-` was also present during the is-rendering phase of the next page.

**Comment by daun** [comment]
That's probably a bug, then. Feel free to submit a PR or create an issue on the plugin repo.

**Comment by mocodev** [comment, marked as answer]
Actually, never mind. It appears to be working, but visually the classes change so fast from the inspector view I thought it wasn't working this way.

**Reply by hirasso** [reply]
@mocodev great to hear that it's working as intended. Would you mind marking your comment as answer?
